riverdriver/riverpgxv5/internal/dbsqlc/pg_misc.sql

```diff
@@ -1,5 +1,15 @@
 -- name: PGAdvisoryXactLock :exec
 SELECT pg_advisory_xact_lock(@key);
 
--- name: PGNotify :exec
-SELECT pg_notify(@topic, @payload);
\ No newline at end of file
+-- name: PGNotifyMany :exec
+WITH topic_to_notify AS (
+    SELECT
+        concat(current_schema(), '.', @topic::text) AS topic,
+        unnest(@payload::text[]) AS payload
+)
+
+SELECT pg_notify(
```

Comment:
Minor, but any chance we could try to by convention omit the newlines between CTE cases? Every time I see a `SELECT` like this with a bare line above it, I read it as a standalone expression before realizing there's CTE(s) up there and I have to read upwards to get the whole context.